instrumentation/http_instrumentation.go

```diff
@@ -19,36 +24,36 @@ type httpMetricsCollector struct {
 }
 
 // newHTTPMetricsCollector creates a new httpMetricsCollector.
-func newHTTPMetricsCollector(reg *prometheus.Registry, hardcodedLabels []string) httpMetricsCollector {
+func newHTTPMetricsCollector(reg *prometheus.Registry) httpMetricsCollector {
 	m := httpMetricsCollector{
 		requestCounter: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
 			Name: "http_requests_total",
 			Help: "Counter of HTTP requests.",
 		},
-			append(hardcodedLabels, "code", "method", "tenant"),
+			[]string{"group", "handler", "code", "method", "tenant"},
```

Comment:
In fact, I think the rest of the labels can also be added to `hardcodedLabels`?
```suggestion
			hardcodedLabels,
```

Reply:
Not exactly. The meaning of hardcoded labels is that they would always have the same values for each handler where they are used. The others will depend on the request and thus aren't hardcoded labels.